keyvalue/file.go

```diff
@@ -33,12 +33,12 @@ type file struct {
 }
 
 type fileData struct {
-	runOnceFileRecord
+	fs   *FS
+	path string // path is stored as the "key", keeping it here is for generating hackpadfs.FileInfo's
+
 	modeOverride    *hackpadfs.FileMode
```

Comment:
A tiny nit: Since the ordering isn't as flexible anymore and gofmt likes to align the names/types, let's delete the blank lines between fields for all of the realigned structs.

The runOnceFileRecord one in particular is a bit weird. In a future PR, I imagine it'd be worth a small refactor to group the sync.Once and value in a common type. Go 1.21's `sync.OnceValue` looks promising.

Reply:
> Since the ordering isn't as flexible anymore [...]

To be clear, the ordering was never flexible - alignment has always mattered...

I had the same thought about `sync.OnceValue`, but I didn't want to update Go too far, partly because the linter catches more things and I've spent way too much time on this already... A future PR sounds good.

I'll delete blank lines.

Reply:
I hear ya, I was referring to the linter gating things now. Less flexible in the eyes of CI, if you will.